### ISSUE TITLE
Ajax request MismatchsenderID

### ISSUE THREAD

**Issue by NilanKent** [phonegap/phonegap-plugin-push#2445]
### Expected Behaviour
Notification successfull
### Actual Behaviour
MismatchsenderID
### Reproduce Scenario (including but not limited to)
in version 1.8 works fine, but i want update to 2.2.3 and i get the mismatch error

i have the config.xml with :
-`<plugin name="phonegap-plugin-push" spec="^2.2.3" />`
-`<resource-file src="google-services.json" target="google-services.json" />`
-`<resource-file src="GoogleService-Info.plist" />`

i have the "google-services.json" and "GoogleService-Info.plist" in root. (same directory like config.xml)
and both was download from general tab from firebase proyect

i have init with:
```
var push = PushNotification.init({
         android: {},
         browser: {
            pushServiceURL: 'http://push.api.phonegap.com/v1/push'
         },
         ios: {
            "sound": true,
            "vibration": true,
            "badge": true
         }
      });
```

i have ajax with:
```
$.ajax({
         type: 'POST',
         url: "https://fcm.googleapis.com/fcm/send",
         headers: {
            Authorization: 'key=AAAAHo[...]Q2z'
         },
         contentType: 'application/json',
         dataType: 'JSON',
         data: JSON.stringify({"registration_ids": JSON Array with ID's, "data": {"title": title, "body": body}}),
         beforeSend: function () {
         },
         success: function (response) {    
           alert("ok");
         },
         error: function (xhr, status, error) {
            console.log(xhr.error);
         }
      });
```
#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)
Any
#### (Android) What device vendor (e.g. Samsung, HTC, Sony...)
Any
#### Cordova CLI version and cordova platform version
cordova (Cordova CLI) : 7.1.0
#### Plugin version
2.2.3

Does anyone know if I miss/forget something?

**Comment by macdonst** [comment]
@NilanKent Uh, `MismatchsenderID` means you are trying to send from one FCM project to a different FCM project. That is your device is registering with a different FCM sender ID than the one your service is using to send the push message. You'll need to double check that the sender ID used in the google services file matches the one your service is using.

**Comment by NilanKent** [comment]
I have reviewed the sender ID many times and I do not see anything strange. It's all right, right? Or is there an error?

Firebase settings
![captura](https://user-images.githubusercontent.com/40631070/42019810-5c453586-7ab6-11e8-83d3-3e753bfb3f02.JPG)

Google-services.json
![captura2](https://user-images.githubusercontent.com/40631070/42019811-5c650e6a-7ab6-11e8-9a7d-a14819c6e58c.JPG)

ajax
![captura3](https://user-images.githubusercontent.com/40631070/42019812-5c7de7b4-7ab6-11e8-8d9d-8b964fd8cacb.JPG)

Should the blue box match? I've tried with both (default and changing it) but does not work either.
For the rest, point 1 and 2 always match. Maybe i'm forgetting any? Maybe the project can not find the file Google-services.json when device is registering?
